OpenKh.Command.IdxImg/Program.cs

```diff
@@ -199,11 +199,24 @@ private class InjectCommand
             [Option(CommandOptionType.NoValue, Description = "Do not compress the file to inject", ShortName = "u", LongName = "uncompressed")]
             public bool Uncompressed { get; set; }
 
-            [Option(CommandOptionType.SingleValue, Description = "ISO block for KH2.IDX. By default is 1840 for KH2FM", ShortName = "idx", LongName = "idx-offset")]
-            public long IdxIsoBlock { get; set; } = 1840;
+            [Option(CommandOptionType.SingleValue, Description = "ISO block for KH2.IDX. Determined automatically by default.", ShortName = "idx", LongName = "idx-offset")]
+            public int IdxIsoBlock { get; set; } = -1;
 
-            [Option(CommandOptionType.SingleValue, Description = "ISO block for KH2.IMG. By default is 671693 for KH2FM", ShortName = "img", LongName = "img-offset")]
-            public long ImgIsoBlock { get; set; } = 671693;
+            [Option(CommandOptionType.SingleValue, Description = "ISO block for KH2.IMG. Determined automatically by default", ShortName = "img", LongName = "img-offset")]
+            public int ImgIsoBlock { get; set; } = -1;
+
+            bool CompareArrays(byte[] input1, byte[] input2)
```

Comment:
No need to create a dedicated function. `input1.SequenceEqual(input2)` will be good enough and it's fast when comparing two arrays of the same length!